### ISSUE TITLE
Economic model in Inner-Ring is not able to process 3000+ containers

### ISSUE THREAD

**Issue by vdomnich-yadro** [nspcc-dev/neofs-node#2173]
If neoFS network contains 3000 containers or more, function [ListContainerSizes](https://github.com/nspcc-dev/neofs-contract/blob/master/container/container_contract.go#L555) fails with error:
```
contract execution finished with state FAULT; exception: at instruction 4249 (LDLOC6): stack is too big
```

This function is used by economic model in Inner-Ring. The error means that economic model will stop working when neoFS network grows to the size of about 3000 containers.

## Possible Solution
If we switch to function [IterateContainerSizes](https://github.com/nspcc-dev/neofs-contract/issues/293) it will be possible to run economic model with arbitrary container size.

## Regression
No

## Your Environment
```
neofs-ir --version
Version: v0.34.0-113-g45c64dbe-dirty 
GoVersion: go1.18.4
```

**Comment by roman-khimov** [comment]
It stops around 2048 in fact, an iterator is needed here and for the `List(owner []byte) [][]byte` as well (I'm not sure where it's used now, but the problem would be the same).

**Comment by roman-khimov** [comment]
Can be fixed with contracts 0.17.0 including nspcc-dev/neofs-contract#293 fix.
